### ISSUE TITLE
pH probe erratic readings

### ISSUE THREAD

**Issue by laurenzane** [Putnam-Lab/CBLS_Wetlab#7]
The pH probe used in the CBLS aquarium with Mettler Toledo probe (order #61343101; [model: InLab Expert Pro](https://www.mt.com/us/en/home/products/Laboratory_Analytics_Browse/pH-meter/sensor/pH-sensor/InLab-Expert.html)) is unresponsive and reads a value of 0.0 mV when submerged in pH 4.01, pH 7.01, and pH 10.01 solutions, thus preventing calibration. Alteratively, aquarium members noted that if the probe is responsive, readings produced can range up to -/+ 500 mV.

Troubleshooting steps:
1. I attempted to calibrate pH meter according to [Orion Meter Maintenance ](https://github.com/Putnam-Lab/Lab_Management/blob/master/Lab_Resources/Equipment_Protocols/Orion_MultiParameter_Sensor_Protocol.md). Calibration failed because readings all read 7.11 pH. 
2. I unpluged and replugged probe, rinsed probe with DI water and replaced storage solution. 
3. Finally, I followed the [manufacturer protocol](https://github.com/Putnam-Lab/CBLS_Wetlab/files/10745574/369913.pdf) to troubleshoot the probe, which failed because the reading stabilized at -235.1 mV in pH 7.00 buffer. 

![IMG-2273](https://user-images.githubusercontent.com/64224939/219102417-e2981f8f-3362-4ce7-8f72-1a23631a9abd.jpg)
![IMG-2272](https://user-images.githubusercontent.com/64224939/219102378-bf79dce3-87c6-4a68-80fb-a836cc3052f7.jpg)

**Comment by laurenzane** [comment]
https://docs.google.com/spreadsheets/d/1YTO4ClqWsxRGtV8Bx3Kz4KibiiSuDW2GgushPaws_m0/edit#gid=0

**Comment by hputnam** [comment]
Please post a picture of the glass bulb on the end of the pH probe.

Also please attach a different probe to test if is it a probe or a meter problem.

**Comment by laurenzane** [comment]
I attached a working pH probe to the Orion Star meter. I recorded measured values (n=3)  in pH 4.01, pH 7.00 and pH 10.01 standard solutions and concluded that the other probe has no issues (see above Google Spreadsheet). I replaced the old probe with the tested probe and put the old probe into the box with other pH probes and taped a sticky note detailing connection issues to the old probe.

![IMG-2753](https://user-images.githubusercontent.com/64224939/233686952-b72ce430-b45c-40b0-ac0e-a831242a44e6.jpg)
